cpp/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ set(UCXX_BUILD_TESTS ${BUILD_TESTS})
 set(UCXX_BUILD_BENCHMARKS ${BUILD_BENCHMARKS})
 set(UCXX_BUILD_EXAMPLES ${BUILD_EXAMPLES})
 
-set(UCXX_CXX_FLAGS "")
+set(UCXX_CXX_FLAGS "-Wall" "-Wattributes" "-Werror" "-Wextra" "-Wsign-conversion" "-Wno-missing-field-initializers")
```

Comment:
You shouldn't need all the quotes here.

Reply:
Thanks for the hint, removed in https://github.com/rapidsai/ucxx/pull/340/commits/f974f2c4a0ab4235c3140fc76fe3e110019f865f .